### ISSUE TITLE
Imported Linked content is not showing up in the ipfs localhost

### ISSUE THREAD

**Issue by GeetaSarvadnya** [brave/brave-browser#16680]
## Description 

Found while testing https://github.com/brave/brave-browser/issues/15937
Imported Linked content is not showing up in the ipfs localhost
## Steps to Reproduce

   1. Clean profile 1.27.x
   2. Enable ipfs and open blog. brave.com
   3. Import a linked content
   4. Imported Linked content is not showing up in the ipfs localhost

## Actual result:

![image](https://user-images.githubusercontent.com/38657976/123840072-8a2da380-d92b-11eb-8284-5f8ee3b28078.png)
![image](https://user-images.githubusercontent.com/38657976/123840131-9a458300-d92b-11eb-8d0d-c1d619645a42.png)

## Expected result:
linked content should be shown instead of object object

## Reproduces how often: 

Always
## Brave version (brave://version info)

Brave | 1.27.84 Chromium: 91.0.4472.124 (Official Build) beta (64-bit)
-- | --
Revision | 7345a6d1bfcaff81162a957e9b7d52649fe2ac38-refs/branch-heads/4472_114@{#6}
OS | Windows 10 OS Version 2004 (Build 19041.1052)

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome? NA

## Miscellaneous Information:

cc: @brave/legacy_qa @spylogsster

**Comment by lidel** [comment]
This is caused by a bug in ipfs-webui.
It was  fixed in latest version shipped with go-ipfs 0.9.0, so only thing here is to retest after we update to go-ipfs 0.9

**Comment by spylogsster** [comment]
lets check this after https://github.com/brave/qa-resources/issues/321

**Comment by GeetaSarvadnya** [comment]
Please note the issue applicable for import `This page` context menu also. 

![image](https://user-images.githubusercontent.com/38657976/123984851-9cb6e400-d9e2-11eb-89a8-442a49d1bc89.png)
![image](https://user-images.githubusercontent.com/38657976/123984924-ac362d00-d9e2-11eb-8ea1-9819a706da2a.png)

**Comment by stephendonner** [comment]
@GeetaSarvadnya mind double-checking if this is fixed with the latest `go-ipfs` `0.10.0-rc1` release (or later) when you get a chance?  Thanks!

**Comment by GeetaSarvadnya** [comment]
Verification PASSED on

```
Brave | 1.34.80 Chromium: 97.0.4692.71 (Official Build) (64-bit)
-- | --
Revision | adefa7837d02a07a604c1e6eff0b3a09422ab88d-refs/branch-heads/4692@{#1247}
OS | Windows 10 Version 21H2 (Build 19044.1415)

```
 - Confirmed that import `Linked content`, `This page` and `Selected text` context menus are working as expected, click on any of the imported files opens up the content of the imported file.
 
![IPFS Linked content import](https://user-images.githubusercontent.com/38657976/148813552-eee9ef28-f0ae-4ea4-93ba-63990b554134.gif)
